cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -311,7 +311,7 @@ struct dispatch_histogram
   template <typename ActivePolicyT,
             typename DeviceHistogramInitKernelT,
             typename DeviceHistogramSweepKernelT>
-  CUB_RUNTIME_FUNCTION _CCCL_FORCEINLINE cudaError_t
+  CUB_RUNTIME_FUNCTION _CCCL_ATTRIBUTE_HIDDEN _CCCL_FORCEINLINE cudaError_t
```

Comment:
Looks good to me, maybe we should find a proper name for the combination of
`CUB_RUNTIME_FUNCTION _CCCL_ATTRIBUTE_HIDDEN _CCCL_FORCEINLINE`